app/controllers/overrides/registrations_controller.rb

```diff
@@ -4,7 +4,7 @@ class RegistrationsController < DeviseTokenAuth::RegistrationsController
     # method copied from original gem; refactored to please Rubocop
     def create
       setup_resource_for_create
-      return render_create_error if name_fields_have_invalid_characters?
+      return render_create_error if name_fields_have_invalid_characters? || !AccountValidationService.valid_dob?(account_params[:DOB])
```

Comment:
We can probably add name_fields_have_invalid_characters and any future validation to the AccountValidationService

Reply:
I moved it to AccountValidationService, can you review that in the PR app to make sure everything is working properly?

Reply:
Looks good!